Mathlib/RingTheory/IntegralClosure/IntegralRestrict.lean

```diff
@@ -360,6 +365,12 @@ lemma Algebra.intNorm_zero : Algebra.intNorm A B 0 = 0 := by
 
 variable {A B}
 
+theorem Algebra.intNorm_eq_of_algEquiv (x : B) (σ : B ≃ₐ[A] B) :
```

Comment:
```suggestion
@[simp]
theorem Algebra.intNorm_map_algEquiv (x : B) (σ : B ≃ₐ[A] B) :
```
is this true for `B ≃ₐ[A] C`?

Reply:
> is this true for B ≃ₐ[A] C?

It should since [Algebra.norm_eq_of_algEquiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Norm/Basic.html#Algebra.norm_eq_of_algEquiv) holds in that generality but the proof uses [galRestrict](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/IntegralClosure/IntegralRestrict.html#galRestrict) which holds only for `B ≃ₐ[A] B`. So I guess it would be necessary first to generalize `galRestrict`. 

BTW, I chose the name `Algebra.intNorm_eq_of_algEquiv` by analogy with `Algebra.norm_eq_of_algEquiv`. I don't mind changing it to `Algebra.intNorm_map_algEquiv` but then I should probably do the same for `Algebra.norm_eq_of_algEquiv`? (and the following `Algebra.norm_eq_*` results too?)